packages/sources/finage/src/config/index.ts

```diff
@@ -6,7 +6,7 @@ export const NAME = 'FINAGE'
 export const DEFAULT_BASE_URL = 'https://api.finage.co.uk'
 export const DEFAULT_STOCK_WS_API_ENDPOINT = 'wss://e4s39ar3mr.finage.ws:7002'
 export const DEFAULT_FOREX_WS_API_ENDPOINT = 'wss://w29hxx2ndd.finage.ws:8001'
-export const DEFAULT_CRYPTO_WS_API_ENDPOINT = 'wss://72x8wsyx7t.finage.ws:6008'
+export const DEFAULT_CRYPTO_WS_API_ENDPOINT = 'wss://e3tne9d5zq.finage.ws:6014'
```

Comment:
Where did this change originate from? I'd like to make sure that this is necessary since it's a source code change...

Reply:
The old one was not working. @cds95 has more info on this

Reply:
Let's please get sign-off from the data team before making this change. @cds95 if you have context on this, will you get confirmation from them?

Reply:
This URL was the URL the data team (Fionn and Will) gave to me.